extensions/amp-story/1.0/amp-story.js

```diff
@@ -1052,6 +1052,22 @@ export class AmpStory extends AMP.BaseElement {
     }, {});
   }
 
+  /**
+   * Reacts to system layer buttons display state.
+   * @param {boolean} canShowButtons
+   * @private
+   */
+  onCanShowSystemLayerButtonsUpdate_(canShowButtons) {
+    if (!this.topBar_) {
```

Comment:
should we add a test?

Reply:
Plan is to move this code to the ``system-layer`` asap. This will, at worst, display an error in the console. But I hate advocating against tests, so if you feel strongly about this I'll add one

Reply:
I don't feel strongly :)